src/index.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import {registerPlugin} from '@playkit-js/playkit-js';
+import {registerPlugin} from 'kaltura-player-js';
```

Comment:
don't we keep the registerPlugin on playkit as well? it'll break other plugins

Reply:
we keep just for backward compatibility. 
our plugins should be aligned with the latest.